docs/src/theme/DocSidebar/Desktop/Content/styles.module.css

```
@@ -20,24 +24,30 @@
   .pane {
     flex-shrink: 0;
-  }
-
-  .rootPaneEntry {
-    animation: root-pane-enter 150ms cubic-bezier(0.175, 0.885, 0.32, 1.1) both;
+    transition:
+      transform var(--sidebar-pane-duration) var(--sidebar-pane-easing),
+      opacity var(--sidebar-pane-duration) var(--sidebar-pane-easing);
   }
 
   .rootPaneInactive {
```

Stylelint (17.14.0)

[error] Expected class selector ".rootPaneInactive" to be kebab-case (selector-class-pattern)

(selector-class-pattern)